datamodels/2.x/itop-oauth-client/en.dict.itop-oauth-client.php

```diff
@@ -93,6 +93,7 @@
 	'Class:OAuthClientAzure/Attribute:used_for_smtp+' => 'At least one OAuth client must have this flag to “Yes”, if you want iTop to use it for sending mails',
 	'Class:OAuthClientAzure/Attribute:used_for_smtp/Value:yes' => 'Yes',
 	'Class:OAuthClientAzure/Attribute:used_for_smtp/Value:no' => 'No',
+    'Class:OAuthClientAzure/Attribute:tenant' => 'Tenant',
```

Comment:
```suggestion
    'Class:OAuthClientAzure/Attribute:tenant' => 'Tenant',
    'Class:OAuthClientAzure/Attribute:tenant+' => '~~',
```

Add a description entry that will be used as a tooltip to help understand where to get it and what kind of value to set.